### PR TITLE
 Add search field element

### PR DESCRIPTION
**Pull request by J-Bannach**

### CODE REVIEW

File: src/app/components/SearchField/SearchField.module.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.label {
+  background: #f3f3f5;
+  color: #023047;
```
Comment (lajanzen) [inline src/app/components/SearchField/SearchField.module.css:3]: Maybe you have some color variables that you can use for your colors in CSS?

**Review by lajanzen — APPROVED** [review]
I can't see your deployment

File: src/app/components/SearchField/SearchField.module.css (hunk 1 above)
Comment (J-Bannach) [reply src/app/components/SearchField/SearchField.module.css:3]: That's correct atm. Will be changed to CSS variables in a later step (started working on this component before my globals CSS were merged). 
Thanks for the advice!

**Review by MaxMuecket — APPROVED** [review]
Nice work! 

Maybe you can add a description ✌️

**Review by Dev-dfm — APPROVED** [review]
lgtm 🚀

**Pull request merged**